packages/kbn-securitysolution-utils/src/path_validations/index.ts

```diff
@@ -52,39 +52,60 @@ export enum OperatingSystem {
 export type EntryTypes = 'match' | 'wildcard' | 'match_any';
 export type TrustedAppEntryTypes = Extract<EntryTypes, 'match' | 'wildcard'>;
 
-export const validateFilePathInput = ({
+export const validateWildcardInput = ({
+  field = '',
   os,
   value = '',
 }: {
+  field?: string;
   os: OperatingSystem;
   value?: string;
 }): string | undefined => {
   const textInput = value.trim();
+  if (field === 'file.path.text') {
+    return validateFilePathInput({ os, value: textInput });
+  }
+  return validatePotentialWildcardInput(textInput);
+};
+
+const validateFilePathInput = ({
+  os,
+  value,
+}: {
+  os: OperatingSystem;
+  value: string;
+}): string | undefined => {
   const isValidFilePath = isPathValid({
     os,
     field: 'file.path.text',
     type: 'wildcard',
-    value: textInput,
+    value,
   });
   const hasSimpleFileName = hasSimpleExecutableName({
     os,
     type: 'wildcard',
-    value: textInput,
+    value,
   });
 
-  if (!textInput.length) {
+  if (!value.length) {
     return FILEPATH_WARNING;
   }
 
   if (isValidFilePath) {
     if (hasSimpleFileName !== undefined && !hasSimpleFileName) {
-      return FILENAME_WILDCARD_WARNING;
+      return WILDCARD_WARNING;
     }
   } else {
     return FILEPATH_WARNING;
   }
 };
 
+const validatePotentialWildcardInput = (value?: string): string | undefined => {
+  if (/\*|\?/.test(value ?? '')) {
```

Comment:
my editor says that using single character substituion is not that great after all. Using a set is better. 😅 
```suggestion
  if (/[*?].test(value ?? '')) {
```